src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs

```diff
@@ -140,6 +158,117 @@ public virtual void InitializeMining()
                 this.miningTask = this.CreateBlocksAsync();
                 this.asyncProvider.RegisterTask($"{nameof(PoAMiner)}.{nameof(this.miningTask)}", this.miningTask);
             }
+
+            // Initialize the interop polling loop, to check for interop contract requests.
```

Comment:
I presume this is a copy paste error from the original PR, because this comment is wrong

Reply:
@zeptin yes this will resolved with the next PR